modules/deploy/pages/deployment-option/cloud/managed-connectors/create-gcp-bigquery-connector.adoc

```diff
@@ -57,7 +57,7 @@ click *Create Connector*.
 | A comma-separated list of the cluster topics you want to replicate to Google BigQuery.
 
 | `Topics regex`
-| A Java regular expression of topics to replicate. For example: specify `.*` to replicate all available topics in the cluster. Applicable only when *Use regular expressions* is selected.
+| A Java regular expression of topics to replicate. For example: specify `.\*` to replicate all available topics in the cluster. Applicable only when **Use regular expressions** is selected.
```

Comment:
```suggestion
| A Java regular expression of topics to replicate. For example: specify `.*` to replicate all available topics in the cluster. Applicable only when **Use regular expressions** is selected.
```
In the deploy preview this backslash is rendered in the code span:
![image](https://github.com/redpanda-data/docs/assets/36839689/b212f8c4-9c2f-4ab0-8c43-4ea2c33ba043)

If the backslash was supposed to escape the asterisk in AsciiDoc, I think just using the double asterisk around "Use regular expressions" works well by itself (this is from my local preview):
![image](https://github.com/redpanda-data/docs/assets/36839689/ba2cb51e-da02-4991-8965-c91215cf0d36)